### PR TITLE
Refactor express server

### LINKED ISSUE #7: Review npm modules

### LINKED ISSUE #12: Refactor Express Server
In order to have the express server code cleaner and better organize, we will refactor what was provided from node-skeleton to achieve the following item:

### Database
- [x] Create .env file with database credentials
- [x] Create db.js to connect to the database (Pool) and export as a function
- [x] Create query files (import db.js) where each query is a function and export
- [x] Create the router files based on REST. (basics to tests the routes only)

### PR DESCRIPTION
**Pull request by wbox**
Closes #12 
Closes #7 

We are going to meet at 1pm to review and merge the this PR.

### CODE REVIEW

File: lib/orders-queries.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  return pool
+    .query('SELECT * FROM orders;')
+    .then(res => res.rows)
+    .catch(err => err.messages)
+};
+
+const getOrderById = (id) => {
+  const values = [ id ];
+  return pool
+    .query('SELECT * FROM orders WHERE id = %1', values)
```
Comment (wbox) [inline lib/orders-queries.js]: Fix % -> $

File: lib/restaurants-queries.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  return pool
+    .query('SELECT * FROM restaurants;')
+    .then(res => res.rows)
+    .catch(err => err.messages)
+};
+
+const getRestaurantById = (id) => {
+  const values = [ id ];
+  return pool
+    .query('SELECT * FROM restaurants WHERE id = %1', values)
```
Comment (wbox) [inline lib/restaurants-queries.js]: % -> $

File: lib/orders-queries.js (hunk 1 above)
Comment (wbox) [reply lib/orders-queries.js]: Fixed

File: lib/restaurants-queries.js (hunk 2 above)
Comment (wbox) [reply lib/restaurants-queries.js]: fixed.

**Review by gabygab159 — APPROVED** [review]

**Review by jacquelinel33 — APPROVED** [review]

**Pull request merged**